simpleviewinc/re-theme
# Fix Rollup exports and css px units

### Updates
  * Added more tests
  * Updated how rollup does the build and export
    * Now exports a cjs version, and an esm
    * Update package.json to use `cjs` for main, and `esm` for module
      * Allows it to be consume as an uncompiled version when needed ( esm / react-native )
      * Allows it to be consume as a compiled version when needed ( cjs / node )
  * Removed adding px to rules on web
    * No requires the consuming app to define the css unit value
  * Remove the docs folder from the repo
    * **That's the main reason there are so many file changes**
    * It's only used for the example app, which get's built and lives on its own branch
  * Cleaned up dimensions on web
    * Also added tests
  * Updated get and join helpers folder path
    * Removed unused code
    * Was doing some caching stuff that never worked how I wanted
    * Makes the build smaller, so that's good
  * Updated readme to match changes
  * Added fixes to scripts and example app deploy
    * Example app is now deployed at => https://simpleviewinc.github.io/re-theme/

### To Test
There are two different ways to test these changes

 1. [Example App](https://simpleviewinc.github.io/re-theme/)
    * Clone this repo
    * Navigate to the app directory
    * Run `yarn install`
    * Run `yarn start`
    * The example app should open up in a web browser

 2. Pull this [Keg-Components](https://github.com/simpleviewinc/keg-components/tree/ZEN-122-keg-components-import-export) branch
    * Update your keg components to pull down this re-theme branch in it's package.json
    * Remove yarn.lock, and reinstall node_modules
    * Run the keg-components storybook => `yarn sb`
